jina/peapods/peas/helper.py

```diff
@@ -92,6 +93,7 @@ def update_runtime_cls(args, copy=False) -> 'Namespace':
         _args.runtime_cls not in gateway_runtime_dict.values()
         and _args.host != __default_host__
         and not _args.disable_remote
+        and not ip_address(_args.host).is_loopback
```

Comment:
what is this?

Reply:
This is checking if the host parameter is actually localhost (aka on the same machiene), in this case we should not use jinad as we only need it for remote entities.

Reply:
yes, but the runtime is actually not removed here, right? I mean not properly at least. So my code will do weird stuff without this check?

Reply:
ah actually I will remove the check completely, you are right